ssg/build_cpe.py

```diff
@@ -257,10 +269,20 @@ def enrich_with_cpe_info(self, cpe_products):
         self.ansible_conditional = cpe_products.get_cpe(self.cpe_name).ansible_conditional
         self.cpe_name = cpe_products.get_cpe_name(self.cpe_name)
 
+    def pass_parameters(self, product_cpes):
```

Comment:
It is pretty difficult to deduce what the method does from its name. And it is not easy to deduce that even from the body. Do we pass parameters to self, or to product CPEs? What parameters do we pass? The one that we call `arg`, or is it about something else? Such questions arise, and they are not easily answered by looking at the code.